src/main/java/net/rpgtoolkit/common/utilities/TileSetCache.java

```diff
@@ -20,7 +27,7 @@
 public class TileSetCache {
 
   // Singleton.
-  private static final TileSetCache instance = new TileSetCache();
+  private static final TileSetCache INSTANCE = new TileSetCache();

```

Comment:
Yes. It'll make it easier to use (just TileSetCache.foo() instead of TileSetCache.getInstance().foo()), but otherwise I'm not aware of anything. Singleton was new to me when I started on this project, but my understanding from reading a bit is that it's not super necessary here, given the way Java does static.

Personally, I think it's easier to think of it as "The tile set cache" rather than, "The tile set cache's instance (which we could make more of, but we shouldn't, so that ability is private)".